### DISCUSSION TITLE
Fake-IP 过滤列表黑名单命中的域名是直接绕过代理流程直接访问还是获取真实IP后再进行匹配？

### DISCUSSION THREAD

**Discussion by lvxj11** [MetaCubeX/mihomo#1622] [Q&A] [answered]
如果是直接绕过代理那我想直接过滤geosite cn域名，让大部分国内域名绕过代理直接访问。
但有个问题就是类似googleapis.cn这种域名会被geosite cn命中但实际需要代理的情况怎么处理？
可不可以在黑名单规则中排除，或者优先白名单再匹配黑名单？

**Comment by fejeIaqbhbv** [comment, marked as answer]
> Fake-IP 过滤列表黑名单命中的域名是直接绕过代理流程直接访问还是获取真实IP后再进行匹配？
> 如果是直接绕过代理那我想直接过滤geosite cn域名，让大部分国内域名绕过代理直接访问。
> 但有个问题就是类似googleapis.cn这种域名会被geosite cn命中但实际需要代理的情况怎么处理？
> 可不可以在黑名单规则中排除，或者优先白名单再匹配黑名单？

fake-ip-filter只管连接的是不是真实ip
怎么连接是rules的问题
